dump_utils.cpp

```diff
@@ -167,6 +167,16 @@ void createPEL(
 {
     try
     {
+        if (additionalData.empty() || additionalData.size() < 3)
+        {
+            log<level::WARNING>(
+                fmt::format("User data map's size(({})) is not sufficient to "
+                            "create a PEL message for dump delete/offload",
+                            additionalData.size())
+                    .c_str());
+            throw std::runtime_error("Dump delete/offload PEL not created due "
+                                     "to insufficient info passed");
+        }
```

Comment:
why this change in this commit shoudn;t this be in first commit

Reply:
Hopefully corrected now. Kindly check and confirm. Sorry for the mess up.